spec/Section 3 -- Type System.md

```diff
@@ -179,20 +179,27 @@ made available via introspection.
 
 To allow GraphQL service designers to easily publish documentation alongside the
 capabilities of a GraphQL service, GraphQL descriptions are defined using the
-Markdown syntax (as specified by [CommonMark](https://commonmark.org/)). In the
+Markdown syntax (as specified by [CommonMark](http://commonmark.org/)). In the
```

Comment:
```suggestion
Markdown syntax (as specified by [CommonMark](https://commonmark.org/)). In the
```

why not https ?

Reply:
Nice catch 👍  I rebased this PR on top of master so it's fixed.